spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/web/ServerProperties.java

```diff
@@ -381,6 +382,19 @@ public Session getSession() {
 		 */
 		private final Resource resource = new Resource();
 
+		/**
+		 * Configuring processor cache.
+		 */
+		private int processorCache = 200;
+
+		public int getProcessorCache() {
```

Comment:
The getter/setter pair should be added in the right order (after the one for `acceptCount`) rather than adding them where they are now.